### PR TITLE
(fix) [SD-2475] Bylines and datelines need to be upper and lower case

### PR DESCRIPTION
**Pull request by sivakuna-aap**

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/styles/themes.less [hunk 1]
```diff
@@ -270,11 +270,14 @@ body, html {
 
 	  	.dateline {
 		  	display: flex;
+            input {
+              text-transform: uppercase;
+            }
+
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/styles/themes.less]: (minor) Can we fix the indentation of this block for better readability?

**Comment by plamut** [comment]
Is there a test for the correct dateline format? IMO this is something worth having.

Looks good though. :+1:

**Comment by sivakuna-aap** [comment]
There are behave tests around dateline but none of them check the format because it has current date. I will check those once again and add one for checking the format if possible.

**Comment by plamut** [comment]
OK, sounds good (but if it is not possible, then we can leave it as is I guess, since it doesn't increase the technical debt).

**Comment by sivakuna-aap** [comment]
@plamut added some unit tests for checking the format

**Comment by plamut** [comment]
@sivakuna-aap Cool!

**Pull request merged**